### ISSUE TITLE
Can "Run" and "Stop" shrink when the screen is narrow?

### ISSUE THREAD

**Issue by schanzer** [brownplt/code.pyret.org#442]
Every time I run PD over Zoom, I am reminded that the Stop button goes rogue and covers part of the Definitions Area when the screen is narrow. 

Can we use a CSS media query to check for super-narrow screens, and shrink the width of those buttons accordingly?

**Comment by asolove** [comment]
Yeah, I will find a way to do something about this. The main issue here is the Run/Type dropdown reserving enough space for the very wide text for the typed label.
